mod/ckeditor/views/default/ckeditor/init.php

```diff
@@ -9,7 +9,7 @@
 }
 ?>
 <script>
-	require(['elgg/ckeditor'], function (elggCKEditor) {
+	require(['elgg/ckeditor/config', 'ckeditor/ckeditor', 'elgg/ckeditor'], function(elggCKEditor) {
```

Comment:
Does ```elgg/ckeditor/config``` get loaded twice, too? With ckeditor.js I could change the src in start.php to ckeditor/ckeditor as you have done in #10753. Then it wouldn't get loaded twice either (I haven't done before as I wanted to change as few as possible in the module structure).

I think the problem might not necessarily be the module structure as such but rather WHEN the dependencies are resolved - and this might be quite a tiny bit different with/without cache symlink (and it might be different even on some systems only!!!). Requiring the modules only within elgg/ckeditor might then result in a race condition under some conditions while the resolving of the modules works when already required in init.php.

Let me update this PR with regards of referring to ckeditor.js. It's not necessarily as much shuffling of modules necessary as done in #10753 (at least to get it working on my test installation).